src/BuildScriptGeneratorCli/Commands/BuildCommand.cs

```diff
@@ -82,18 +88,85 @@ public string LanguageVersion
             }
         }
 
-        [Option(
-            "-o|--output <dir>",
-            CommandOptionType.SingleValue,
-            Description = "The destination directory.")]
         public string DestinationDir { get; set; }
 
-        [Option(
-            OptionTemplates.ManifestDir,
-            CommandOptionType.SingleValue,
-            Description = "The path to a directory into which the build manifest file should be written.")]
         public string ManifestDir { get; set; }
 
+        public static Command Export()
+        {
+            var logOption = new Option<string>(OptionTemplates.Log, OptionTemplates.LogDescription);
+            var debugOption = new Option<bool>(OptionTemplates.Debug, OptionTemplates.DebugDescription);
+            var sourceDirArgument = new Argument<string>("SourceDir", "The source directory.");
```

Comment:
nit: I think it would be worth introducing a new `ArgumentTemplates` file (similar to `OptionTemplates`) that can store the hard-coded strings referenced on this line

Reply:
For this one, i think hardcoded argument would be ok because unlike multiple options, there is only one argument throughout all the commands.